draft-ietf-lamps-csr-attestation.md

```diff
@@ -640,13 +776,28 @@ The current registry values can be retrieved from the IANA online website.
 
 # Security Considerations
 
+In the RATS architecture, when Evidence or an Attestation Result is presented to a Relying Party (RP), the RP may learn detailed information about the Attester unless that information has been redacted or encrypted. Consequently, a certain amount of trust must be placed in the RP, which raises potential privacy concerns because an RP could be used to track devices. This observation is noted in Section 11 of {{RFC9334}}.
+
+Typically, the RPs considered in the RATS architecture are application services that use remote attestation, rather than RAs or CAs. Devices inherently place significant trust in RA/CA infrastructure elements, and therefore any additional information revealed through remote attestation to such entities is generally less concerning than disclosure to application services. The problem of copying Evidence by CAs into an X.509 certificate is discussed in {{sec-con-publishing-x509}}.
+
+These privacy risks can be mitigated using several approaches, including:
```

Comment:
Differential privacy analysis may be applied to Evidence, Attestation Results, or Certificates to reveal personally identifiable information that may be used to track a user or to correlate user's transactions.

Reply:
There is a lot that could be done but I did not want to provide an exhaustive list